system-tests-new/tiny-project/test_tiny_project.py

```diff
@@ -0,0 +1,56 @@
+from pathlib import Path
+
+from fab.steps.walk_source import FindSourceFiles
+
+from fab.build_config import BuildConfig
+from fab.steps.analyse import Analyse
+from fab.steps.compile_fortran import CompileFortran
+from fab.steps.grab import GrabFolder
+from fab.steps.link_exe import LinkExe
+from fab.steps.preprocess import fortran_preprocessor
+
+
+class TestTinyProject(object):
+
+    def test(self, tmp_path):
+        # We want to build in a clean workspace each time the test is run.
+        # (Pytest provides the tmp_path fixture.)
+        self.fab_workspace = tmp_path
+        print(f"fab_workspace is {self.fab_workspace}")
```

Comment:
Do we want to be printing out of the tests?

Reply:
That print is useful when a test is failing and we want to investigate the artefacts.

(Also, it was suggested that we pull out these tests into a separate PR, which I've done here: https://github.com/bblay/fab/pull/12.)